.github/stale.yml

```diff
@@ -0,0 +1,16 @@
+daysUntilStale: 90
+daysUntilClose: 30
+exemptLabels:
+  - discussion
+  - security 🔒
+  - "enhancement :sparkles:"
+staleLabel: stale
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity. It will be closed if no further activity occurs. Thank you
+  for your contributions.
```

Comment:
This is what React uses:

> This issue has been automatically marked as stale. **If this issue is still affecting you, please leave any comment** (for example, "bump"), and we'll keep it open. We are sorry that we haven't been able to prioritize it yet. If you have any new additional information, please include it with your comment!

Maybe let's follow their lead and be a bit more gentle, e.g.

```suggestion
  This issue has been automatically marked as stale. **If this issue is still affecting you, please leave any comment** (for example, "bump"), and we'll keep it open. If you have any new additional information—in particular, if this is still reproducible in the [latest version of Electron](https://www.electronjs.org/releases/stable) or in the [beta](https://www.electronjs.org/releases/beta)—please include it with your comment!
```